### ISSUE TITLE
Create resources failed probabilistically when executing command "azd up" or "azd pipleline config"

### ISSUE THREAD

**Issue by Menghua1** [fangjian0423/todo-java-postgresql-terraform#3]
**Describe the issue:**
When excuting command `azd up --template https://github.com/fangjian0423/todo-java-postgresql-terraform -l "eastus"` or `azd pipeline config`, the following error will occur **probabilistically**. 
We executed it five times and encountered this error twice, details are as follows:
![image](https://user-images.githubusercontent.com/111940661/220291571-57f08eea-00fe-47b4-be96-52770086070c.png)

**Repro steps:**
1. Execute command `azd login`.
2. Execute command `azd up --template https://github.com/fangjian0423/todo-java-postgresql-terraform -l "eastus"` .

**or**
1. Execute command `azd login`.
2. Execute command `azd up --template https://github.com/fangjian0423/todo-java-postgresql-terraform -l "eastus"` .
3. Execute command `azd pipeline config`.
4. Click on the link that is printed to open the actions page.

**Environment:**
- OS: windows 
- Template: `https://github.com/fangjian0423/todo-java-postgresql-terraform`
- Branch: main.
- Azd version: azd version 0.6.0-beta.3-daily.2200320 (commit b9ac6ad2da8910a02e54468415f24d84c1255b9c)

**Expected behavior:**
The commands `azd up --template https://github.com/fangjian0423/todo-java-postgresql-terraform -l "eastus"` and `azd pipeline config` can be successfully executed **stably**.

@fangjian0423 for notification.

**Comment by fangjian0423** [comment]
I think Azure Database for PostgreSQL - Flexible Server is not stable for the AAD admin setting in terraform, it's not a template issue.

**Comment by fangjian0423** [comment]
Retry can resolve it, let me add the case on readme **Troubleshooting** section.

**Comment by v-xuto** [comment]
Template `todo-java-postgresql-aca-terraform` has the same issue.

**Comment by fangjian0423** [comment]
> Template todo-java-postgresql-aca-terraform has the same issue.

Yes, it's not template's issue. The postgresql flexible server AAD admin setting in terraform is not stable. Sometimes, it works fine in my local.
